openrtb_ext/imp_gamma.go

```diff
@@ -0,0 +1,27 @@
+package openrtb_ext
+
+// ExtImpGamma defines the contract for bidrequest.imp[i].ext.gamma
+type ExtImpGamma struct {
+	PartnerID string `json:"id"`
+	ZoneID    string `json:"zid"`
+	WebID     string `json:"wid"`
+	AppID     string `json:"app_id,omitempty"`
+	AppName   string `json:"app_name,omitempty"`
+	AppBundle string `json:"app_bundle,omitempty"`
+}
+
+// id=1397808490&
```

Comment:
Are these the values that a publisher is suppose to use in their requests or they are just sample values?

Reply:
`PartnerID`, `ZoneID` and `WebID` are required, the others are unnecessary, we've removed them.